ci-operator/step-registry/ipi/azure/arcconformance/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- fabianofranz
```

Comment:
find a collaborator, so you can go on vacation?

Reply:
Ah I see what you mean by "vacation". :-) I'll add some more people.

Reply:
Done.